src/helpers/singHelper.ts

```diff
@@ -0,0 +1,67 @@
+export function getPitchFromMidi(midi: number): string {
+  const mapPitches: Array<string> = [
+    "C",
+    "C#",
+    "D",
+    "D#",
+    "E",
+    "F",
+    "F#",
+    "G",
+    "G#",
+    "A",
+    "A#",
+    "B",
+  ];
+  const pitchPos = midi % 12;
+  return mapPitches[pitchPos];
+}
+
+export function getDoremiFromMidi(midi: number): string {
+  const mapPitches: Array<string> = [
+    "ド",
+    "ド",
+    "レ",
+    "レ",
+    "ミ",
+    "ファ",
+    "ファ",
+    "ソ",
+    "ソ",
+    "ラ",
+    "ラ",
+    "ラ",
```

Comment:
- 最後が「ラ」になってます、おそらく「シ」だと思います！
```suggestion
    "シ",
```

Reply:
修正済みです(特に問題ないかと思うのでResolveします)